### PR TITLE
Ability to Scroll to the top of the page

### PR DESCRIPTION
**Pull request by pratikmishra356**
I had only changed the index.html file the rest three files were not changed.

#169

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,7 +88,25 @@
...
+         }
+
+  // When the user clicks on the button, scroll to the top of the document
+       function topFunction() {
+       document.body.scrollTop = 0; // For Safari
+       document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
+       }
       </script>
 
+    <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
```
Comment (brenoferreira) [inline index.html]: This file shouldn't contain any app logic. I think this code is better placed at the [SearchGrid component](https://github.com/creativecommons/cccatalog-frontend/blob/master/src/components/SearchGrid.vue)

Comment (pratikmishra356) [reply index.html]: It means we have to change this on SearchGrid.vue file to enable this scroll while searching??

Comment (brenoferreira) [reply index.html]: Yes. That component is where this code would be best placed I think

Comment (pratikmishra356) [reply index.html]: Yes I am trying to place this code on Searchgrid.vue

**Pull request closed without merging**